### ISSUE TITLE
Embed contents of relative urls

### ISSUE THREAD

**Issue by Denperidge** [Denperidge-Redpencil/divio-docs-parser#2]
This is a nobrainer for things like SVG's, but might even be useful for Markdown?

**Comment by Denperidge** [comment]
Alt text should be kept for images. Other markdown files should perhaps be added in a details box?

**Comment by Denperidge** [comment]
Confirmed markdown embed in 880d9b351ac07baf3f1f189ee9693e86cb3970fe !

**Comment by Denperidge** [comment]
Confirmed svg embed in a38e0323ac2898f7172d45fe445dbe4f723fdeec !

**Comment by Denperidge** [comment]
And image embed with 06af36109134df3f80568a17a4de2349c0596716 ! Issue done!
